pkg/skaffold/runner/build_deploy.go

```diff
@@ -100,17 +100,18 @@ func (r *SkaffoldRunner) Build(ctx context.Context, out io.Writer, artifacts []*
 
 // Test tests a list of already built artifacts.
 func (r *SkaffoldRunner) Test(ctx context.Context, out io.Writer, artifacts []graph.Artifact) error {
+	eventV2.TaskInProgress(constants.Test, r.devIteration)
 	if err := r.tester.Test(ctx, out, artifacts); err != nil {
+		eventV2.TaskFailed(constants.Test, r.devIteration, err)
```

Comment:
Note: This will not add event per tester right? We can add those later in a separate PR.

Reply:
Yup, this is just for the over arching test phase